iothub/service/src/DigitalTwin/DigitalTwinClient.cs

```diff
@@ -34,12 +34,66 @@ public static DigitalTwinClient CreateFromConnectionString(string connectionStri
             return new DigitalTwinClient(iotHubConnectionString.HttpsEndpoint, sharedAccessKeyCredential, handlers);
         }
 
+
         private DigitalTwinClient(Uri uri, IotServiceClientCredentials credentials, params DelegatingHandler[] handlers)
         {
-            _client = new IotHubGatewayServiceAPIs(uri, credentials, handlers);
+#pragma warning disable CA2000 // Dispose objects before losing scope (objects are disposed of when this client is disposed)
+#if NETCOREAPP && !NETCOREAPP2_0 && !NETCOREAPP1_0 && !NETCOREAPP1_1
+            // SocketsHttpHandler is only available in netcoreapp2.1 and onwards.
+            HttpMessageHandler httpMessageHandler = new SocketsHttpHandler();
+#else
+            HttpMessageHandler httpMessageHandler = new HttpClientHandler();
+#endif
+            // Currently, the connection lease timeout isn't configurable in this client. This may be worth adding later though
+            ServicePointHelpers.SetLimits(httpMessageHandler, uri, ServicePointHelpers.DefaultConnectionLeaseTimeout);
```

Comment:
Since there isn't a great way to add the option to configure this connection lease timeout, I'll just leave it as the default for now. We can always add a constructor later that takes this as a configurable option if a customer asks for it